### ISSUE TITLE
Formatter duplicates an HTML comment after `{@render}` in Svelte, doubling it on every run

### ISSUE THREAD

**Issue by ToP29** [biomejs/biome#11541]
### Environment information

```
Biome 2.5.10
Windows 11, Node 22.14.0
```

### Description

With `html.experimentalFullSupportEnabled`, the Svelte formatter duplicates an HTML comment that follows a `{@render ...}` tag. Every `biome format --write` run **doubles** the number of copies, so the file grows exponentially.

`biome format` (check mode) reports the file as needing formatting every time, and `biome check` never reports anything wrong — the growth is silent.

### Steps to reproduce

`biome.json`:

```json
{
	"formatter": { "enabled": true, "indentStyle": "tab" },
	"html": {
		"experimentalFullSupportEnabled": true,
		"formatter": { "enabled": true, "indentScriptAndStyle": true }
	}
}
```

`E.svelte`:

```svelte
<div>
	{@render children?.()}
	<!-- comment -->
</div>
```

Run `biome format --write E.svelte` three times.

### Expected result

Formatting is idempotent — the file is unchanged after the first run.

### Actual result

```svelte
<div>
	{@render children?.()}
	<!-- comment -->
	<!-- comment -->
	<!-- comment -->
	<!-- comment -->
	<!-- comment -->
	<!-- comment -->
	<!-- comment -->
	<!-- comment -->
</div>
```

Line counts across runs: 4 → 5 → 7 → 11 → 19 → 35 …

### Notes

- The comment content is irrelevant — a plain `<!-- comment -->` is enough.
- `{@render ...}` immediately before the comment is what triggers it. The same comment without a preceding `{@render}` formats idempotently.
- This was found in a real component library, where the affected file had grown from 425 to 24,977 lines over several `--write` runs before anyone noticed. Because the duplicated text sat inside a comment, the Svelte compiler ignored it, so tests and the production build kept passing.
- Possibly related but distinct: #10236 covers `/* */` comments inside `<script>` getting progressively re-indented; this one duplicates content in the markup.

**Comment by dyc3** [comment]
playground: https://biomejs.dev/playground/?tab=formatter&pane=Diagnostics&language=svelte#code=PABkAGkAdgA%2BAAoACQB7AEAAcgBlAG4AZABlAHIAIABjAGgAaQBsAGQAcgBlAG4APwAuACgAKQB9AAoACQA8ACEALQAtACAAYwBvAG0AbQBlAG4AdAAgAC0ALQA%2BAAoAPAAvAGQAaQB2AD4A

next time, please provide reproductions in one of our preferred formats. we have issue templates.
